docs/setup/configuring-reporting.asciidoc

```diff
@@ -20,9 +20,13 @@ to make sure your hosts support our latest build of Chromium. For the most relia
 installing {kib} using <<docker, Docker>>, or using <<set-up-on-cloud, Elastic Cloud>>.
 ============
 
-For security, you grant users access to the {report-features} and secure the reporting endpoints
-with TLS/SSL encryption. Additionally, you can install graphical packages into the operating system
-to enable the {kib} server to have screenshotting capabilities.
+For security, you can grant users access to the reporting features and secure the reporting endpoints with {ref}/remote-clusters-cert.html[TLS encryption]. Additionally, you can install graphical packages into the operating system to enable the {kib} server to have screenshotting capabilities.
+
+[NOTE]
+============
+
+In 8.19.0, API keys are used to authenticate requests to generate reports. If you have a cross-cluster search environment and want to generate reports from remote clusters, you must have the appropriate cluster and index privileges on the remote cluster and local cluster. For example, if requests are authenticated with an API key, the API key requires certain privileges on the local cluster that contains the leader index, instead of the remote. For more information and examples, refer to {ref}/remote-clusters-cert.html#remote-clusters-privileges-ccr[Configure privileges for cross-cluster replication].
```

Comment:
```suggestion
In 8.19.0, API keys are used to authenticate requests to generate reports. If you have a cross-cluster search environment and want to generate reports from remote clusters, you must have the appropriate cluster and index privileges on the remote cluster and local cluster. For example, if requests are authenticated with an API key, the API key requires certain privileges on the local cluster that contains the leader index, instead of the remote. For more information and examples, refer to {ref}/remote-clusters-privileges-cert[Configure roles and users for remote clusters].
```